### PR TITLE
S11 product info

### PR DESCRIPTION
**Pull request by sofiiadorosh**

### CODE REVIEW

File: src/components/AboutProduct/index.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import { ReactComponent as Plus } from '@assets/plus.svg';
+import { CountPicker } from '@components/CountPicker';
+import { Stars } from '@components/Stars';
+import { useAppSelector } from '@hooks';
+import { selectProductDetails } from '@store/productDetails/selectors';
+import { getNewPrice } from '@utils';
+
+import './AboutProduct.scss';
+
+export enum Buttons {
```
Comment (Yuliia143) [inline src/components/AboutProduct/index.tsx]: 1.Bad naming. 
2.Do you export it from component somewhere ? Then it shouldn't be here

File: src/components/AboutProduct/index.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+          {color && (
+            <li className="details__item">
+              <span className="details__item_color_grey">Color:</span>
+              <span>{color}</span>
+            </li>
+          )}
+          <li className="details__item">
+            <span className="details__item_color_grey">Delivery area</span>
+            <span>{deliveryArea}</span>
+          </li>
```
Comment (Yuliia143) [inline src/components/AboutProduct/index.tsx]: Don't you want to generate an object  ?

File: src/components/AboutProduct/index.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        </ul>
+        <div>
+          <div className="details__price-info">
+            <div className="details__price">
+              <span>{getTotalPrice()} USD</span>
+              <span className="details__price_old">
+                {(price[unit] * count).toFixed(2)} USD
```
Comment (Yuliia143) [inline src/components/AboutProduct/index.tsx]: Logic

File: src/store/products/selectors.ts [hunk 4]
```diff
@@ -18,15 +19,12 @@ export const selectIsLoading = (state: RootState) => state.products.isLoading;
...
 export const selectError = (state: RootState) => state.products.error;
 
-export const selectProductDetails = (state: RootState) =>
-  state.products.selectedProduct;
-
 export const selectPrices = createSelector([selectProducts], products => {
-  let min = products[0].price.new;
-  let max = products[0].price.new;
+  let min = getNewPrice(products[0].price.pcs, products[0].discount);
+  let max = getNewPrice(products[0].price.pcs, products[0].discount);
```
Comment (Yuliia143) [inline src/store/products/selectors.ts]: 2 the same values ?

Comment (sofiiadorosh) [reply src/store/products/selectors.ts]: It doesn't matter what the initial values are; the main purpose is to infer the type as number

File: src/components/AboutProduct/index.tsx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+              ({reviews?.length} customer review)
+            </span>
+          </div>
+        </div>
+        <p className="details__description">{overview}</p>
+        <ul className="details__list">
+          {Object.entries(description).map(([key, value]) => {
+            if (key === 'Color' && !value) {
+              return null;
+            }
```
Comment (Yuliia143) [inline src/components/AboutProduct/index.tsx]: Logic

**Review by Yuliia143 — APPROVED** [review]

**Pull request merged**